### PR TITLE
Mark service.version resource attribute as stable.

### LINKED ISSUE #1901: Add OTEL_SERVICE_VERSION
Similarly to OTEL_SERVICE_NAME, I was wondering if we _could_ add this one as well (for context, we at Lightstep leverage this and currently have to use our own custom env var).

I do remember a conversation back in the day (I think @bogdandrutu and @Oberon00 chimed in) when it came to what made `OTEL_SERVICE_NAME` special, and its siblings not (service namespace, service version). Happy to discuss!

### PR DESCRIPTION
**Pull request by carlosalberto**
Includes adding the respective OTEL_SERVICE_VERSION environment variable. Fixes #1901 

This PR:

* Adds the `service.version` to the `system` resource attributes with environment variable section (which is marked as stable).
* Because of the previous point, we mark the `service.version` attribute itself as stable too.

We very recently marked `service.name` as stable, and I wonder what would be the requirements to do the same for this one - or, put in other way, what clarifications or spikes we would need to do. The only one I can think of is the semantics (currently very lax).

cc @jsuereth

### CODE REVIEW

File: specification/resource/semantic_conventions/README.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ These are the attributes which MAY be configurable via a dedicated environment v
 as specified in [OpenTelemetry Environment Variable Specification](../../sdk-environment-variables.md):
 
 - [`service.name`](#service)
+- [`service.version`](#service)
```
Comment (lmolkova) [inline specification/resource/semantic_conventions/README.md:58]: would you mind marking it (along with `service.name`) as stable in [yaml](
https://github.com/open-telemetry/opentelemetry-specification/blob/1f0478d9bac0be3749dc8903263a521c4cd89aa3/semantic_conventions/resource/service_experimental.yaml#L38)? 

```yaml
      - id: version
        stability: stable
```

BTW, it's also possible to change md generation script to show stable label next to it by adding `--md-stable` arg to the cmd line at 

https://github.com/open-telemetry/opentelemetry-specification/blob/1f0478d9bac0be3749dc8903263a521c4cd89aa3/Makefile#L77-L78

which will make it look like this:

| `service.version` | string | **Stable**<br>The version string of the service API or implementation. | `2.0.0` | Recommended |


or, with` --md-stable --md-use-badges`,  it'll render a cool badge instead: 

| `service.version` | string | Stable<br>The version string of the service API or implementation. | `2.0.0` | Recommended |

Comment (lmolkova) [reply specification/resource/semantic_conventions/README.md:58]: oh, sorry, the `--md-stable` has to wait until https://github.com/open-telemetry/build-tools/pull/170 - currently `stability` defaults to `stable`  and everything will be marked as stable if we render it this way

**Review by reyang — APPROVED** [review]
LGTM with a changelog entry.

File: specification/resource/semantic_conventions/README.md [hunk 2]
```diff
@@ -76,6 +77,7 @@ as specified in the [Resource SDK specification](../sdk.md#sdk-provided-resource
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
 | `service.name` | string | Logical name of the service. [1] | `shoppingcart` | Required |
+| `service.version` | string | The version string of the service API or implementation. | `2.0.0` | Recommended |
```
Comment (joaopgrassi) [inline specification/resource/semantic_conventions/README.md:80]: Shouldn't we add this in https://github.com/open-telemetry/opentelemetry-specification/blob/main/semantic_conventions/resource/service.yaml?

**Comment by reyang** [comment]
@carlosalberto heads up - most likely this PR will be closed, and we'll ask you to resubmit the PR in a new repo, please refer to https://github.com/open-telemetry/opentelemetry-specification/pull/3474#issuecomment-1539386505.

**Pull request closed without merging**